src/Synth/OscilGen.h

```diff
@@ -79,6 +84,8 @@ class OscilGenBuffers : NoCopyNoMove
     int    oscilprepared;   //1 if the oscil is prepared, 0 if it is not prepared and is need to call ::prepare() before ::get()
 
     float hmag[MAX_AD_HARMONICS], hphase[MAX_AD_HARMONICS]; //the magnituides and the phases of the sine/nonsine harmonics
+
+    std::atomic<unsigned> m_change_stamp;
```

Comment:
Would you mind clarifying the multi-threaded context you're expecting this variable to be modified in?

Reply:
I was expecting that this variable might be accessed both by RT and non-RT. After all, it is increased after the change of *any* parameter. However, assuming that *all* parameters are non-RT, I think it is not required to keep this variable `atomic`.

Reply:
Reading more of my own code, I think `atomic` is correct.

Of course, every change of any parameter will change `m_change_stamp`, and this is done by MiddleWare. On the other hand, we have `ADnoteParameters::requestWavetables`, which reads the change stamp in order to do comparisons ("has the OscilGen been changed recently?"). I think this implies using `std::atomic`.

Reply:
Excellent, that's the info I was trying to figure out. Mind dropping ADnoteParameters::requestWavetables and the relevant MiddleWare method in a comment next to the variable?